### ISSUE TITLE
WCAG: Parsing

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#14]
En samling av feil på kodestruktur som enten er brudd på WCAG eller som ikke følger best practice. 

### Feil plassering av `<style>`
`<style>` Skal ikke være plassert i `<body>`
![Skjermbilde 2022-04-11 kl  08 43 59](https://user-images.githubusercontent.com/33222679/162734366-d8e11481-5274-4edb-bef2-c7816852c712.png)
- [x] plasser all `<style>` i `<head>`
-----

### Bruk av prosentverdier i `width` for img
Om man bruker prosenter for å beskrive størrelsen på et bilde skal dette ikke bli lagt i `width` attributtet for `img` elementet. Men heller løses med CSS. [Beskrivelse av feil](https://rocketvalidator.com/html-validation/bad-value-x-for-attribute-width-on-element-img-expected-a-digit-but-saw-instead)
- [x] Når apputvikler bruker prosentverdier blir dette ikke lagt på `width`
-----

### for attributtet peker til en `<div>`
I grupper med radioknapper eller sjekkbokser har vi en label som er plassert i en legend. Denne skal beskrive hele gruppen. I dag så peker **for** attributtet på en` <div> `isteden for en `<input>`. En gruppe med radioknapper eller sjekkbokser vil ha flere input, det kan være man kan peke på flere. Eventuelt så skal man få riktig oppførsel om man legger tekstverdien rett i legend/fieldset som inneholder alle inputer. Da trenger man ikke forholde seg til label/for problematikken. [Eksempel](https://www.accessibility-developer-guide.com/examples/forms/grouping-with-fieldset-legend/#grouping-certain-kind-of-controls)

- [ ] Grupper med radio/sjekkbokser er satt opp riktig

**Comment by haakemon** [comment]
Style i header er fikset i https://github.com/Altinn/app-template-dotnet/pull/75

Cc: @lorang92

**Comment by Febakke** [comment]
Closing this issue. The last task should be handled in #753 and is still relevant
